acceptance/go.mod

```diff
@@ -9,8 +9,8 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/gruntwork-io/terratest v0.46.7
 	github.com/hashicorp/consul-k8s/control-plane v0.0.0-20240226161840-f3842c41cb2b
-	github.com/hashicorp/consul/api v1.28.2
-	github.com/hashicorp/consul/proto-public v0.6.0
+	github.com/hashicorp/consul/api v1.28.4
+	github.com/hashicorp/consul/proto-public v0.6.1
```

Comment:
I think I left it alone because it looked like in the 1.18 branch proto-public was at 0.6.1, but it's an easy bump. I left it as 0.6.1 in control-plane,cli and acceptance to match the 1.18 branch, but can def bump to 0.6.2 if we think that's the right thing. I think it's probably fine either way because its the same version 1.18.x is on atm

Reply:
Cool, yeah hopefully we’re mostly at latest on all the branches after this extended series of PRs 😅 likely not critical if you run into issues.